modules/releng.gradle

```diff
@@ -343,6 +344,80 @@ prepareMajorIncrement {
 	}
 }
 
+tagAppReleases {
+	description = "Gathers the Git IDs from all app changelog files and tags them as 'marketplace-appName-appVersion'."
+
+	doLast {
+		FileTree appChangelogFileTree = fileTree(dir: projectDir, includes: [".releng/apps/**/app.changelog"])
+
+		appChangelogFileTree.each {
+			appChangelog ->
+
+			String appName = appChangelog.getParentFile().getName()
+
+			String appChangelogTextFiltered = appChangelog.filterLine {
+				it =~ /app\.git\.id/
+			}
+
+			List appGitIDLines = appChangelogTextFiltered.readLines()
```

Comment:
Instead of `ID`, can you call it `Id`? I think we always use it that way, like in `userId`.